cypress/utils/helpers.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const apiRequest = (url, method = 'POST', body = undefined, qs = undefined) =>
+  cy.request({
+    method: method,
+    failOnStatusCode: false,
+    url: url,
+    headers: {
+      'content-type': 'application/json',
+      'osd-xsrf': true,
+    },
+    body: body,
+    qs: qs,
+  });
+
+export const devToolsRequest = (url, method = 'POST', body = undefined, qs = undefined) =>
+  cy.request({
+    method: 'POST',
+    form: false,
```

Comment:
nit: seems like there could be a request headers constant that we can spread here